### ISSUE TITLE
epoch<500就过拟合了？

### ISSUE THREAD

**Issue by lixiangamy** [wentaozhu/adversarial-deep-structural-networks#8]
我在DDSM训练了300就出现了过拟合
INbreast训练500就出现了过拟合

**Comment by wentaozhu** [comment]
If you want to train more epochs, you can use smaller learning rate.
